node/src/components/chain_synchronizer/operations.rs

```diff
@@ -537,13 +537,16 @@ async fn fast_sync_to_most_recent(
     chainspec: &Chainspec,
     node_config: NodeConfig,
 ) -> Result<(KeyBlockInfo, BlockHeader), Error> {
+    info!("start - fetch the last switch block before the trusted block - fast sync");
     let mut trusted_key_block_info =
         get_trusted_key_block_info(effect_builder, chainspec, &trusted_block_header).await?;
+    debug!("finish - fetch the last switch block before the trusted block - fast sync");
```

Comment:
Why not log at `INFO` just like the start message?

Reply:
I thought it was unnecessarily noisy (https://github.com/casper-network/casper-node/pull/2637#discussion_r796833292). Happy to change it back if you disagree.

Reply:
No, that's fine. I figured out (too late) that reading timestamps of `start ...` logs is enough to figure out the timeline.